exercises/alice_russell/ex02_oo_basics/src/Shape.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
This works well in Visual Studio but it's not what you'll see in Mantid as it's not guaranteed to work in all of our environments. In Mantid you'll see the use of 
```c++
#ifndef SHAPE_H_
#define SHAPE_H_

... class and function definitions

#endif //SHAPE_H_